changelog.txt

```diff
@@ -17,6 +17,7 @@
 * Fix - Multi-Currency settings rounding option and preview.
 * Fix - Payment failure on checkout block with UPE when phone number field is hidden
 * Add - Add compatibility between Multi-Currency and WooCommerce UPS shipping extension.
+* Add - Add SEPA Debit to available UPE payment methods.
```

Comment:
Let's make this more exciting!

> * Add - *Early access*: allow your store to collect payments with Giropay and Sofort. Enable the feature in settings!

This is the way the changelog entry looks for Giropay and Sofort.